src/wp-includes/class-wpdb.php

```diff
@@ -2414,11 +2414,14 @@ public function log_query( $query, $query_time, $query_callstack, $query_start,
 	 *
 	 * @since 4.8.3
 	 *
+	 * @param bool $reset Optional. Whether to reset the placeholder. Default false.
```

Comment:
```suggestion
	 * @param bool $reset Optional. Whether to reset the placeholder. Default false.
	 *
```
Also needs a `since` annotation below the `4.8.3` entry.